addons/mail/static/tests/chat_window/chat_window_new_message_tests.js

```diff
@@ -71,6 +71,30 @@ QUnit.test("fold", async (assert) => {
     assert.containsOnce($, ".o-mail-ChannelSelector");
 });
 
+QUnit.test("new message search should display no result when no matches found", async (assert) => {
+    await start();
+    await click(".o_menu_systray i[aria-label='Messages']");
+    await click(".o-mail-MessagingMenu-new");
+    await insertText(".o-mail-ChannelSelector", "Rainbow Panda");
+    assert.containsOnce($, ".o-mail-ChannelSelector-suggestion:contains(No results found)");
+});
+
+QUnit.test(
+    "new message search should not be visible when clicking outside of the field",
+    async (assert) => {
+        const pyEnv = await startServer();
+        const partnerId = pyEnv["res.partner"].create({ name: "Panda" });
+        pyEnv["res.users"].create({ partner_id: partnerId });
+        await start();
+        await click(".o_menu_systray i[aria-label='Messages']");
+        await click(".o-mail-MessagingMenu-new");
+        await insertText(".o-mail-ChannelSelector", "Panda");
+        assert.containsOnce($, ".o-mail-ChannelSelector-suggestion");
+        await click(".o-mail-ChatWindow-content");
+        assert.containsNone($, ".o-mail-ChannelSelector-suggestion");
+    }
+);
+
```

Comment:
Better keep number of tests low. Chat window coverage is somewhat redundant with discuss app, so removing these tests.